src/controllers/admin.js

```diff
@@ -0,0 +1,31 @@
+import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
+import dbClient from '../utils/dbClient.js'
+
+export const updateUserRoleById = async (req, res) => {
+  const id = Number(req.params.id)
+  const { content } = req.body
+  const foundUserRole = req.user.role
+
+  const foundUser = await dbClient.user.findUnique({
+    where: { id, content }
+  })
```

Comment:
What's user.content? I don't think that exists

Reply:
Removed content from foundUser